prac_05/word_occurrences.py

```diff
@@ -0,0 +1,22 @@
+"""
+Word Occurrences
+Estimate: 20 minutes
+Actual:   28 minutes
+
+"""
+
+word_to_count = {}
+text = input("Text: ")
+words = text.split()
+for word in words:
```

Comment:
You can sort the words list on this line by:
for word in sorted(words):

This saves you having to do it later on line 16.

Reply:
Good idea